pkg/v3/types/basetypes.go

```diff
@@ -1,8 +1,11 @@
 package types
 
 import (
+	"encoding/json"
 	"fmt"
 	"math/big"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
```

Comment:
We don't want to import chain specific packages in this repo here unfortunately

Reply:
I wonder is there a way we can add a deny list for packages we want to prohibit in keepers 🤔

Reply:
that was my understanding, but then i noticed we're still using go-ethersum in several places in the codebase (although it looks like these are all under "sim" - which I assume is only used in testing?

will remove this instance!

Reply:
yep, sim is just for testing